docs: add 1.8 release notes (#7164)
influxdata/docs-v2: content/influxdb3/explorer/release-notes/_index.md

docker pull influxdata/influxdb3-ui
```

## v1.8.0 {date="2026-04-29"}

#### Features

- **Streaming data subscriptions (beta)**: Stream data into Explorer from MQTT, Kafka, and AMQP sources with field mapping, message format configuration, and contextual help for each provider.
- **Line protocol preview**: Get a live schema preview and inline parse-error surfacing while writing line protocol, so problems show up before you submit.

> - **Query history and saved queries**: Improved history viewer with query metadata, with surfaced ability to save, manage, and re-run named queries from the query tool.

Reviewer: ```suggestion
- **Query history and saved queries**: Browse query history with query metadata, then save, manage, and re-run named queries from the query tool.
```

Makes it imperative like the other bullets and reduces use of "surface".

- **Interactive sample data**: Browse a redesigned sample data page with dataset descriptions, line protocol preview, and schema mapping.
- **Custom sample data with AI**: Generate custom sample datasets from a natural-language prompt to quickly write familiar sample data into a database.
- **Live sample data plugins**: Configure and run processing engine plugins that continuously generate live sample data, with editable trigger arguments and advanced options.
- **Retention period management**: Set, update, or clear retention periods on both databases and individual tables. Duration fields accept flexible humantime values.
- **Write-to-Explore navigation**: Jump directly from a successful write to the Data Explorer to immediately query the data.

#### Bug fixes

- **Line protocol writes**: Improve handling of write size limits with clearer error messages.
- **Triggers**: Fix trigger name resolution and trigger-to-plugin mapping in the plugins UI.

> - **Errors**: Improve error handling and surface clearer messages for failed InfluxDB requests.

Reviewer: ```suggestion
- **Errors**: Improve error handling and output clearer messages for failed InfluxDB requests.
```
